test/steps/CommonSteps.scala

```diff
@@ -250,6 +251,18 @@ Scenario: providing several book suggestions
     await(cache.removeAll())
   }
 
+  Given("""^the file system store the file "([^"]*)"$"""){ (path:String, content:String) =>
```

Comment:
Delete this step, as it duplicate the step ` Given("""^the file "([^"]*)"$""") { (path: String, content: String) =>`